python/satyaml/CTIM.yml

```diff
@@ -0,0 +1,13 @@
+name: CTIM
+norad: 99402
+data:
+  &tlm Telemetry:
+    telemetry: ctim_70cm
```

Comment:
The telemetry definition is called `ctim` rather than `ctim_70cm` in `python/telemetry/__init__.py` and `python/telemetry/ctim.py`. The names need to match. Maybe `ctim_70cm` is a better name in case support for the S-band telemetry gets eventually added (and also for consistency with `cute_70cm`)?